src/request/components/request-view.scss

```diff
@@ -377,54 +377,96 @@
 }
 
 .tab-logs {
-    .fa-svg {
-        height: 12px;
-        width: 12px;
-        margin-right: 5px;
-        vertical-align: middle;
+    .tab-logs-message-count {
+        font-family: 'Segoe UI', 'Selawik Semibold', Tahoma, Geneva, Verdana, sans-serif;
+        font-size: 175%;
     }
-}
 
-.tab-logs-data-replaced-region {
-    font-weight: bold;
-}
+    .tab-logs-table {
+        table-layout: fixed;
+        
+        .fa-svg {
+            height: 12px;
+            width: 12px;
+            margin-right: 3px;
+            vertical-align: middle;
+        }
 
-.tab-logs-message-count {
-    font-family: 'Segoe UI', 'Selawik Semibold', Tahoma, Geneva, Verdana, sans-serif;
-    font-size: 175%;
-}
+        .tab-logs-table-message {
+            display: flex;
+            flex-direction: row;
+            
+            .tab-logs-table-message-icon {
+                visibility: hidden;               
+            }
 
-.tab-logs-data-default {
-    font-weight: normal;
-    color: #333333; // grey
-    vertical-align: middle;
-}
+            .tab-logs-table-message-icon-expandable {
+                @extend .tab-logs-table-message-icon;
+                
+                visibility: visible;               
+            }
 
-.tab-logs-data-warning {
-    @extend .tab-logs-data-default;
-    
-    color: #D35400; // orange
-    
-    .fa-svg {
-        .fa-svg-path {        
-          fill: #D35400; // orange
+            &:hover {
+                .tab-logs-table-message-icon {
+                    visibility: visible;               
+                }
+            }
+ 
+            .tab-logs-table-message-object {
+                overflow: auto;
+                
+                > pre {
+                        margin: 0;
+                        
+                        > code {
+                            background: transparent;
+                            padding: 0;
+                        }
+                    }
+            }
```

Comment:
Have a look at this - https://smacss.com/book/applicability. I don't think its worth changing this now, but before things are through we will need to cleanup our css a bit... note this isn't targeted at this specific code, but I'm guilty of not practicing what I preach here. Hence when we get the new shell design through, we will try and start off on the right foot.